SpatialGDK/Build/Scripts/BuildWorker.bat

```diff
@@ -1,22 +1,69 @@
 @echo off
 
-rem Generate worker configs
-pushd "%~dp0..\..\..\..\..\..\spatial"
-spatial worker build build-config
-popd
+rem Usage: <GameName> <Platform> <Configuration> <game.uproject> [-nocompile] <Additional UAT args>

```

Comment:
What output does a user get if they execute this script with not params?

Reply:
Added some parameter checking which prints if incorrect number of args are passed in